issuance/issuance.go

```diff
@@ -42,31 +42,31 @@ type ProfileConfig struct {
 	AllowSCTList    bool
 	AllowCommonName bool
 
-	Policies            []PolicyInformation
+	Policies            []PolicyInformation `validator:"required"`
```

Comment:
For slices, `required` is the same as `min=1`, right? Maybe we should stick with `min=1` for consistency. Also, it seems like we should apply `dive` to all slices of structs (and maps of structs). Otherwise we could wind up with an unpleasant surprise if we later add validation tags to that struct and find they are not respected.

Reply:
Good catch.